daemon/logger/fluentd/infomap.go

```diff
@@ -0,0 +1,48 @@
+package fluentd
+
+import (
+	"strings"
+
+	"github.com/docker/docker/daemon/logger"
+)
+
+const defaultInfoKeys = "container_id,container_name"
+
+// InfoMap returns information about a container as a map.
+func infoMap(info logger.Info) map[string]string {
```

Comment:
I still don't think we should do this. We can build the map statically in `Log()`, or even when the logger is created as it is not dynamic.

Reply:
This all works today without this mapping.
This is going from a strong type to a weak one.
The logger still needs to know about the keys and everything. So what's the benefit of the added complexity?
This is adding more complexity, but I don't really see the benefit.

Reply:
Can you be a bit more specific on what you mean with "this all works today" and where you see the additional complexity? The current approach is not configurable in any way, so some additional complexity was to be expected, I think.

I can also remove the configurable part again, if that is preferred, although it was a suggestion from your team...

Reply:
> Can you be a bit more specific on what you mean with "this all works today"

Loggers already use these fields today. Why do we need to change it to add a new field?

> where you see the additional complexity

This whole mapping is extra complexity.
We have structured data with predefined keys which should not be opaque to anything accessing this data.